src/plugins/README.md

```diff
@@ -174,3 +165,76 @@ const plugins = [ClausePlugin()];
 * `onClauseUpdated` : `Function` (Required) - Called when the text of a clause changes
 * `pasteToContract` : `Function` (Required) - Loads a template via copy/paste
 * `clauseProps`: (Required) - Props passed to the `ClauseComponent`
+
+### What it returns?
+
+```js
+{
+    name,
+    augmentSchema,
+    renderBlock,
+    isEditable,
+    onChange,
+    onPaste,
+    queries: {
+      findClauseNode,
+      isOutsideOfClause: isEditable,
+      isClauseSupported
+    }
+}
+```
+
+where augmentSchema, renderBlock, isEditable, onChange, onPaste, findClauseNode, isOutsideOfClause, and isClauseSupported are functions.
+
+## ConditionalPlugin
+
+### What it does?
+
+A plugin to support variables corresponding to conditional blocks in templates enabling text which depends on a value of a Boolean variable in the model.
+
+### Usage
+
+```shell
+npm install @accordproject/cicero-ui
+```
+
+```js
+import { ConditionalPlugin } from '@accordproject/cicero-ui';
+import { Editor } from 'slate-react'
+
+const plugins = [ConditionalPlugin()];
+
+<Editor
+  ...
+  plugins={plugins}
+/>
+```
+
+### What it returns
+
+```js
+{
+  name,
+  augmentSchema,
+  isEditable,
+  renderInline,
+}
+```
+
+where augmentSchema, isEditable and renderInline are functions.
```

Comment:
Change to "where `augmentSchema`, `isEditable`, and `renderInline ` are functions."

Reply:
Done